Conversion/MultiCopy.FCMacro

```diff
@@ -0,0 +1,1439 @@
+        )
+    else:
+        msgbox_ = QtGui.QMessageBox(
+            QtGui.QMessageBox.Warning, "MultiCopy - Warning Message", message
+        )
+    msgbox_.setWindowModality(QtCore.Qt.ApplicationModal)
+    msgbox_.exec_()
+
+
+
+# Sub-main Function Class
+# ------------------------------------------------------------------------------------------------
+
+class Filter(QtCore.QObject):
+    """
+    This class has only one function which is responsible for validating 
+    the input paste code commands when the user focusses out of the input 
+    text box. Appropriately, the GUI interface may change by blocking access 
+    to certain GUI elements if the validation returns a false value.
+    """
+    def eventFilter(self, widget, event):
+        if event.type() == QtCore.QEvent.FocusIn:
+            widget.moveCursor(QtGui.QTextCursor.End, QTextCursor.MoveAnchor)
+            mcObj._d(6, "mc_d_label_3").setStyleSheet("background-color:#1c1c1c")
+            mcObj._d(1, "mc_d_paste_button").setEnabled(True)
+            mcObj.validated = False
+            return False
+        elif event.type() == QtCore.QEvent.FocusOut:
+            if mcObj._validate():
+                mcObj._d(6, "mc_d_label_3").setStyleSheet("background-color:darkgreen")
+            else:
+                mcObj._d(6, "mc_d_label_3").setStyleSheet("background-color:crimson")
+                mcObj._d(1, "mc_d_paste_button").setEnabled(False)
+            return False
+        else:
+            return False
+
+
+
+# Roman Numerals Conversion
+# Reference :  https://pypi.org/project/roman/
+# ------------------------------------------------------------------------------------------------
+
+romanNumeralMap = (
+    ("M", 1000),
+    ("CM", 900),
+    ("D", 500),
+    ("CD", 400),
+    ("C", 100),
+    ("XC", 90),
+    ("L", 50),
+    ("XL", 40),
+    ("X", 10),
+    ("IX", 9),
+    ("V", 5),
+    ("IV", 4),
+    ("I", 1),
+)
+
+def IntToRoman(n_):
+    result = ""
+    try:
+        for numeral, integer in romanNumeralMap:
+            while n_ >= integer:
+                result += numeral
+                n_ -= integer
+    except Exception:
+        result = ""
+    return result
+
+def RomanToInt(inputStr_):
+    result = 0
+    index = 0
+    try:
+        for numeral, integer in romanNumeralMap:
+            while inputStr_[index : index + len(numeral)] == numeral:
+                result += integer
+                index += len(numeral)
+    except Exception:
+        result = 0
+    return result
+
+
+
+# Alphabet Conversion
+# ------------------------------------------------------------------------------------------------
+
+alphaMap = [
+    RANDOMSTRING,
+    "a",
+    "b",
+    "c",
+    "d",
+    "e",
+    "f",
+    "g",
+    "h",
+    "i",
+    "j",
+    "k",
+    "l",
+    "m",
+    "n",
+    "o",
+    "p",
+    "q",
+    "r",
+    "s",
+    "t",
+    "u",
+    "v",
+    "w",
+    "x",
+    "y",
+    "z",
+]
+
+def IntToAlpha(n_):
+    result = ""
```

Comment:
Missing docstring for public function. Moreover, the content looks much more complex than what the function name suggests. Maybe another name would fit better. For sure a few example use would help to understand what to expect from the function.

Reply:
Done, with examples!